sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java

```diff
@@ -413,7 +409,6 @@ public void deleteEntity(String partitionKey, String rowKey, String eTag) {
      *                                    table, or if {@code eTag} is not {@code null} and the existing entity's eTag
      *                                    does not match that of the provided entity.
      * @throws IllegalArgumentException if the provided partition key or row key are {@code null} or empty.
-     * @throws RuntimeException if the provided timeout expires.
```

Comment:
Is there no exception thrown anymore? What happens if the timeout expires?

Reply:
Right, so this is interesting. In Tables, operations that work at the Entity level accept a "timeout" parameter that can be sent to the service, where as Table level operations do not. The server also enforces certain limitations on the timeout value itself, such it will be capped to a max value that varies depending on the operation. If the server-side timeout expires, the server will return an error. I haven't seen this happen in practice and it's not documented specifically in the REST api docs or swagger so I'm not sure what error code is returned in this case, vs. other failures.

I will have to do some research and then document the correct type of exception the customer receives when the server-side timeout expires, but I'll file a separate issue for that.